### DISCUSSION TITLE
Submission for ppl using google colab

### DISCUSSION THREAD

**Discussion by tony-teng-yt377** [Part-II-Computational-Physics/.github#14] [General]
Hi, I have finished the tasks with Google Colab Jupiter notebook, how am I suppose to submit it? (In the instruction it says I should create a repository of a copy of advection_... notebook and make changes to it, but for me I had trouble opening it in any editor (VS for example), so I decided to do the task in google colab Jupiter notebook. ) Would appreciate specific instruction on submission in that format!

**Comment by tony-teng-yt377** [comment]
I have added my notebook to the repository and don't know if that would be fine? (Coz it is super hard to edit the VS notebook given, since it is kinda in text format)

**Comment by NathanMagnan** [comment]
So in theory at the begining of the week you should have created a copy of the online repository on your machine, and worked on the jupyter notebook that was in this copy. Then at the end of the week, a simple `git commit` `git push` would been enough to submit your work.

What you have done should work, I guess, as long as "added my notebook to the repository" means "push" not "commit" (you need your new file to be uploaded for us to receive it for grading).

**Reply by tony-teng-yt377** [reply]
<img width="1543" alt="Screenshot 2023-02-22 at 08 51 23" src="https://user-images.githubusercontent.com/125870751/220569763-440adf13-d906-42e3-972d-5cb444febc2f.png">

So that's what my repository look like, would this be fine?

**Reply by AustenLamacraft** [reply]
This looks fine. You might want to try out the [GitHub starter course](https://classroom.github.com/a/w9VOOLL8) to get a sense of what Git and GitHub are really about

**Reply by tony-teng-yt377** [reply]
Yes will have a look, thank you so much!
